test/unit/specs/fields/fieldRadios.spec.js

```diff
@@ -64,31 +66,77 @@ describe("FieldRadios.vue", function() {
 			expect(isChecked(6)).to.be.false;
 		});
 
-		it("radioList value should be the model value after changed", (done) => {
-			model.skills = "ReactJS";
-			vm.$nextTick( () => {
-				expect(isChecked(0)).to.be.false;
-				expect(isChecked(1)).to.be.false;
-				expect(isChecked(2)).to.be.false;
-				expect(isChecked(3)).to.be.false;
-				expect(isChecked(4)).to.be.false;
-				expect(isChecked(5)).to.be.true;
-				expect(isChecked(6)).to.be.false;
-				done();
+		it("label with checked input should have a 'is-checked' class", () =>{
+			expect(labelList[0].classList.contains("is-checked")).to.be.false;
+			expect(labelList[1].classList.contains("is-checked")).to.be.true;
+			expect(labelList[2].classList.contains("is-checked")).to.be.false;
+			expect(labelList[3].classList.contains("is-checked")).to.be.false;
+			expect(labelList[4].classList.contains("is-checked")).to.be.false;
+			expect(labelList[5].classList.contains("is-checked")).to.be.false;
+			expect(labelList[6].classList.contains("is-checked")).to.be.false;
+		});
+
+
+		describe("test values reactivity to changes", () => {
```

Comment:
I have grouped the test a little bit, is this okay, and can I do the same thing in `checklist`'s tests ?